app/views/propertyLinking/declaration.scala.html

```diff
@@ -51,7 +51,7 @@
 
 @propertyStillOccupied = @{request.ses.propertyOccupancy.exists(_.stillOccupied)}
 
-@strippedFileName = @{request.ses.uploadEvidenceData.fileInfo.map(_.name.replaceAll("^([^-]+)-", ""))}
+@strippedFileName = @{request.ses.uploadEvidenceData.fileInfo.map(_.name.fold("")(_.replaceAll("^([^-]+)-", "")))}
```

Comment:
I guess this is because filename can now be None? But it probably should never be None.